### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nazaruha**
Logger and Config logic.

### CODE REVIEW

**Review by diabeticMike — CHANGES_REQUESTED** [review]
shit

File: main.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+func GetValues(w http.ResponseWriter, r *http.Request) {
+	io.WriteString(w, fmt.Sprintln(ListOfPlayers))
+}
+
+// func main() {
+// 	logger.Logger()
+// 	config.Config()
+// }
```
Comment (diabeticMike) [inline main.go]: Don't left redundant comments

File: config/config.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		err = json.NewDecoder(reader).Decode(&config)
+	}
+	if err != nil {
+		log.Printf("Error while reading configuration from JSON (%s) error: %s\n", filePath, err.Error())
+	} else {
+		log.Printf("Configuration from JSON (%s) provided\n", filePath)
+	}
+
+	return err, config
+}
```
Comment (diabeticMike) [inline config/config.go]: rename this file to envconfig.go

**Review by diabeticMike — CHANGES_REQUESTED** [review]
Overall ok, but i think you need to add config.json file with your configuration

File: main.go (hunk 1 above)
Comment (nazaruha) [reply main.go]: DONE!

File: config/config.go (hunk 2 above)
Comment (nazaruha) [reply config/config.go]: DONE!

Comment (diabeticMike) [reply config/config.go]: revert dir name to config pls

**Review by diabeticMike — CHANGES_REQUESTED** [review]
.

**Review by diabeticMike — APPROVED** [review]

**Pull request merged**